scss/inc/_navigatorFizzy.scss

```diff
@@ -0,0 +1,312 @@
+@import 'inc/bootstrap';
+
+$lineHeight: 3rem;
+$borderWidth: 1px;
+$backgroundColor: #f2f2f2;
+$radiusCircular: 50%;
+$sizeDefault: 4rem;
+$hitboxSize: 6.25rem;
+$borderThin: 0.125rem;
+$colorTextDisabled: hsl(0, 0%, 45%);
+$colorBgDefault: white;
+$borderMedium: 0.25rem;
+$borderMediumPlus: 0.375rem;
+$colorTextDefault: hsl(0, 0%, 12%);
+$colorBgInverted: $colorTextDefault;
+$colorTextInverted: white;
+$colorPrimaryHighlight: hsl(208, 100%, 32%);
+$colorSuccess: hsl(164, 54%, 25%);
+$colorWarning: hsl(356, 65%, 40%);
```

Comment:
Do we really have green and red in this UI?

Reply:
No) Missed part of LDS styles